test/specs/global.deprecations.tests.js

```diff
@@ -1,4 +1,80 @@
 describe('Deprecations', function() {
+	describe('Version 2.7.0', function() {
+		describe('Chart.Controller.update(duration, lazy)', function() {
+			var chart;
+			var addAnimationSpy;
+
+			beforeEach(function() {
+				chart = acquireChart({
+					type: 'doughnut',
+					data: {
+						labels: ['A', 'B', 'C', 'D'],
+						datasets: [{
+							data: [10, 20, 30, 100]
+						}]
+					},
+					options: {
+						cutoutPercentage: 85,
```

Comment:
Is this option relevant? we should avoid setting data/options that are not useful for the test